generate-stats.js

```diff
@@ -0,0 +1,159 @@
+const https = require('https');
+const fs = require('fs');
+const maxRetry = 3;
+let retries = 0;
+
+var SI_SYMBOL = ["", "k", "M", "G", "T", "P", "E"];
+
+function abbreviateNumber(number){
+	const tier = Math.log10(number) / 3 | 0;
+
+	if (tier == 0) {
+		return number;
+	}
+
+	const suffix = SI_SYMBOL[tier];
+	const scale = Math.pow(10, tier * 3);
+
+	const scaled = number / scale;
+
+	return scaled.toFixed(1) + suffix;
+}
+
+function httpsPost({body, ...options}) {
+	return new Promise((resolve,reject) => {
+		const req = https.request({
+			method: 'POST',
+			...options,
+		}, res => {
+			const chunks = [];
+
+			res.on('data', data => chunks.push(data))
+			res.on('end', () => {
+				let body = Buffer.concat(chunks);
+
+				switch(res.headers['content-type']) {
+					case 'application/json':
+					try {
+						body = JSON.parse(body);
+					} catch (e) {
+						console.log('Failed to parse reponse, retrying...');
+						if (retries < maxRetry) {
+							retries += 1;
+							console.log('Attempt ' + retries);
+							postRequest()
+						}
+					}
+					break;
+				}
+
+				resolve(body)
+			})
+		})
+		req.on('error', reject);
+
+		if (body) {
+			req.write(body);
+		}
+
+		req.end();
+	})
+}
+
+function generateTemplate(data) {
```

Comment:
When viewing this on a mobile-sized screen, some of the rows skip some spaces.

![Screenshot from 2019-04-17 19-55-27](https://user-images.githubusercontent.com/12687192/56330721-f2c74600-614e-11e9-9596-fb48872fded8.png)

Reply:
Just noticed, it's a very fitting profile picture to be there with the rest of the row missing :laughing: